rpc/src/v1/impls/parity.rs

```diff
@@ -138,6 +138,22 @@ impl<C, M, S: ?Sized, U> Parity for ParityClient<C, M, S, U> where
 		)
 	}
 
+	fn hardware_accounts_info(&self) -> Result<BTreeMap<String, BTreeMap<String, String>>, Error> {
```

Comment:
Could be `BTreeMap<H160, BTreeMap<String, String>>` or even better `BTreeMap<H160, HwAccountInfo>`